src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Random rnd = new Random();
+        Colors color = new ColorSupplier().getRandomColor();
```

Comment:
Do not create a new Instance of `Random` and `ColorSupplier` every time, make it a class-level variable